workflow/hpxml_inputs.json

```diff
@@ -1962,6 +1962,13 @@
     "hvac_control_heating_weekday_setpoint": "64, 64, 64, 64, 64, 64, 64, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 64, 64",
     "hvac_control_heating_weekend_setpoint": "64, 64, 64, 64, 64, 64, 64, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 70, 64, 64"
   },
+  "sample_files/base-hvac-air-to-air-heat-pump-1-speed-lockout-temperatures-geb-multistage-backup.xml": {
```

Comment:
Please remove all references to "geb". But more importantly, I would like to see fewer test files, 9 new test files feels overkill. And that's on top of 6 new test files for the defrost model. Can we just add a single "base-hvac-foo-research-features.xml" for each unique HVAC type that enables all the advanced features (defrost, deadband, multistage backup) at the same time? That would hopefully cut the 15 test files roughly in half.